deploy/kubernetes/overlays/alpha/controller_add_resizer.yaml

```diff
@@ -0,0 +1,17 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-resizer
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-resizer:canary
```

Comment:
We should work on publishing our own resizer image

Reply:
yeah, I can contact releng about this. Lets track seperatly from this PR though